src/web/app/pages-instructor/instructor-search-page/comment-result-table/comment-result-table.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FeedbackSession, QuestionOutput } from "../../../../types/api-output";
+
+/**
+ * Data object for communication with child comment result component
```

Comment:
When writing documentation, it is a good practice to mention what it is but not how it going to be used (because it is already revealed in the code).